src/main/java/com/jcabi/github/RtPull.java

```diff
@@ -159,6 +161,42 @@ public void merge(
     }
 
     @Override
+    public MergeState merge(
+        @NotNull(message = "message can't be NULL") final String msg,
+        final String sha) throws IOException {
+        final JsonObjectBuilder builder = Json.createObjectBuilder()
+            .add("commit_message", msg);
+        if (sha != null) {
```

Comment:
I decided to allow ```null``` SHA here so that one can merge without a SHA and still get possibly more meaningful errors than a simple IOException. Since Java doesn't support return-type only overloading, I couldn't create a ```MergeState merge(String)``` method, as that would have resulted in a collision. Thoughts?